.github/workflows/integration-tests.yml

```diff
@@ -38,6 +38,7 @@ jobs:
         with:
           environment-file: environment_py3_${{ matrix.os-short }}.yml
           environment-name: py3_parcels
+      - run: pip install -r requirements-dev.txt
```

Comment:
The rest of install is doen through conda; why use pip here? Would it not be more consistent to use condo for these packages too? I have some poor experiences with mixing condo and pip...

Reply:
It would be more consistent, but I'm afraid that not all development dependencies will be available through conda channels.

I doubt this would cause any issues as these are pure Python dependencies (and don't require compilers etc). Even if issues arise, it would only be for developers and not users